lib/internal/test_runner/tap_parser.js

```diff
@@ -753,12 +753,9 @@ class TapParser extends Transform {
       }
     }
 
-    const reason = StringPrototypeTrim(this.#readNextLiterals());
-    if (todoOrSkipToken) {
-      if (reason) {
-        test.reason = reason;
-      }
-
+    if (todoOrSkipToken === 'todo' || todoOrSkipToken === 'skip') {
```

Comment:
Nothing that needs to change in this PR, but this whole block (lines 745-761 with the changes in this PR) looks like it could be optimized a bit.